main.tf

```diff
@@ -94,6 +94,13 @@ resource "google_pubsub_topic" "topic" {
       allowed_persistence_regions = message_storage_policy.key == "allowed_persistence_regions" ? message_storage_policy.value : null
     }
   }
+  dynamic "schema_settings" {
+    for_each = var.schema_settings
```

Comment:
This is going to loop over the values in `schema_settings` - probably not what you expected.

Please add this to an example and test to make sure your logic is correct.

Reply:
Hey! Change schema_settings to be an object

Reply:
Hey @morgante,  any feedback on the new changes?

Reply:
@morgante Can we please review an merge this?